type-o-matic/Program.cs

```diff
@@ -95,7 +104,7 @@ static CSFile GeneratePInvokesFromTypes (TypeAggregator types, PlatformName plat
 				.And (new CSUsing ("System"))
 				.And (new CSUsing ("System.Collections.Generic"))
 				.And (new CSUsing ("SwiftRuntimeLibrary"));
-		
+
```

Comment:
Less try and do not add noise in the diff.

Reply:
Done